examples/plasticity/DruckerPrager_Verification.xml

```diff
@@ -0,0 +1,190 @@
+<!--
+Problem: borehole under far-field stress, drained condition
+Created by: Jian Huang
+Last update: 15 Jun 2021
+Objective: to verify the plasticity solver in GEOSX against analytical solution
+Reference: Chen, S. L., & Abousleiman, Y. N. (2017). Wellbore stability analysis using strain hardening and/or softening plasticity models. International Journal of Rock Mechanics and Mining Sciences, 93, 260-268.
+-->
+
+
```

Comment:
```suggestion
<?xml version="1.0" ?>
```

Reply:
This is probably why the xml does not pass the CI check

Reply:
Good to know that